tests/pipeline/test_adjusted_array.py

```diff
@@ -442,6 +548,22 @@ def test_multiplicative_adjustments(self,
                     datetime64ns_dtype,
                 ),
             ),
+            _gen_overwrite_1d_array_adjustment_case(
+                'float',
+                make_input=as_dtype(float64_dtype),
```

Comment:
could we just pass the dtype in, the rest of the arguments can be infered from that.